src/coreclr/tools/dotnet-pgo/MethodMemoryMap.cs

```diff
@@ -67,25 +94,28 @@ internal class MethodMemoryMap
                 MethodDesc method = null;
                 try
                 {
-                    method = idParser.ResolveMethodID(e.MethodID);
+                    method = idParser.ResolveMethodID(e.MethodID, throwIfNotFound: false);
                 }
                 catch
                 {
                 }
```

Comment:
Is the try-catch still necessary?

Reply:
Sadly, I'm not confident that it never throws. I'm just confident it doesn't throw much with that flag set.